### ISSUE TITLE
This is the wrong place to implement flashlight handling

### ISSUE THREAD

**Issue by UniversalSuperBox** [ubports/indicator-power#28]
indicator-power currently handles turning device flashlights on and off in https://github.com/ubports/indicator-power/blob/xenial/src/flashlight.c. To do this, it finds a valid sysfs path from its list of paths and writes a brightness value into it.

This does not seem like an appropriate place to handle finding and setting these values. It would be preferable to have a service which uses IPC to ask to enable and disable the flashlight. This service could better handle per-device quirks than this indicator.

**Comment by Fuseteam** [comment]
does [utorch](https://code.launchpad.net/~majster-pl/utorch/trunk) work on all devices? could that implementation be used to make a service?

**Comment by dobey** [comment]
> It would be preferable to have a service which uses IPC to ask to enable and disable the flashlight.

Right. It seems like `repowerd` would be the most appropriate place to implement the low level part.
